news/_posts/2019-05-29-severity-classification.md

```diff
@@ -0,0 +1,103 @@
+- Difficulty to globally place and manage orders
+
+A critical issue should result in a patch version that should be released as soon as possible. Our recent [spam attack](http://build.prestashop.com/news/fighting-against-spamming-again/) is a good example: it was a critical security issue as our users were flooded with fake customers’ accounts. As a consequence, a patch version has been released that resolve it on 1.6 and 1.7, a good example of a community work.
+
+
+### Major
+
+The bug affects major functionality or major data and there is a workaround (a temporary method for dealing with an issue until a more permanent solution is found), but it is not obvious or can be difficult to put in practice.
+
+A major issue affects a large percentage of users (> 30% and < 60%) and matches at least one of the following:
+
+- It impacts law compliance
+- It has a strong impact on the usability of the front-office / back-office or blocks another project
+- It is an important problem but not necessarily blocking the main activity of the seller
+
+Examples:
+
+- Being unable to add, configure or delete a theme or a module
+- Difficulty in operating a module properly
+- Impacts the price the customer pays
+
+### Minor
+
+The bug affects minor functionality or non-critical data and there is a reasonable workaround, even if it can be annoying when using your shop.
+
+
+Examples:
+
+- A tolerable slowdown
+- A display problem that prevents you to do something (you can’t click on an element)
+- An alert message displayed in your back-office that it is possible to close
+- The product doesn't get added to cart after deleting product customisation
+- When cloning a product, carriers information is not copied
+- Statistics graph not working or showing wrong data 
+
+
+### Trivial
+
+The bug doesn’t affect any functionality or data. It does not impact productivity or efficiency. It is only an inconvenience without functional impact and it does not even need a workaround.
+
+Examples:
+
+- Cosmetic issues
+- Wrong translation in a specific language: that can be solved on [Crowdin](https://crowdin.com/project/prestashop-official)
+- Missing confirmation message after an action
+- A link opened in the same tab instead of a new tab
+
+## Issue Prioritization
+
+Assessing severity helps to prioritize issues but it is not the only criterion at stake. Given two equally severe issues, how to choose one over the other ?
+
+At PrestaShop, prioritization is done by representatives of the Development team, the Product Management team, and the Quality Assurance team. Together, during regular meetings, we look at the new confirmed issues and we sort them.
+
+One of the things we value a lot is making sure that a given bug does not besmirch PrestaShop’s image nor it affects the confidence merchants can have in PrestaShop. This factor is really important as it can durably damage the adoption of the project and its long-term survival. 
```

Comment:
```suggestion
One of the things we value a lot is making sure that a given bug does not damage PrestaShop’s image nor it affects the confidence merchants can have in PrestaShop. This factor is really important as it can durably damage the adoption of the project and its long-term survival. 
```
I think not everybody knows this word

Reply:
Good idea

Reply:
+1